config/locales/en.yml

```diff
@@ -46,11 +46,18 @@ en:
       activity: Sign-in attempts and changes to account details
       description: <p class="govuk-body">Read the <a class="govuk-link" href="https://www.gov.uk/government/publications/govuk-accounts-trial-full-privacy-notice-and-accessibility-statement" data-module="gem-track-click" data-track-category="account-manage" data-track-action="security" data-track-label="privacy-notice">full GOV.UK accounts privacy notice</a> to find out how we collect, process and use information about you.</p>
       event:
-        change_email: Changed email address
-        change_password: Changed password
-        change_phone: Changed phone number
-        login: Signed in
-        report_suspicious_activity: Report suspicious activity
+        account_locked: Password verification failed too many times, account locked for 1 hour.
+        email_change_confirmed: Change of email address has been confirmed.
+        email_changed: Changed email address
```

Comment:
I think this one might need some content design, since the email hasn't actually changed at this point

Reply:
Could make historic data a little confusing, so I think there'll also need to be a migration to turn previous "change requested" events into "change confirmed" events.

Reply:
Actually, no need for a migration.  The `EMAIL_CHANGED` event will be ID 1, the `EMAIL_CHANGE_REQUESTED` event will be the new one.

Reply:
I've decided to make event 1 (pre-existing email change event) say "Changed email address" for now, as it already said that, and made event 11 (new email confirmed event) not show up on the security page, until the collection of messages as a whole can be reviewed.